OmniKitTests/PodInfoTests.swift

```diff
@@ -343,27 +343,125 @@ class PodInfoTests: XCTestCase {
+            XCTAssertEqual(0x14203180, decoded.pulseLogEntry[42])
+            XCTAssertEqual(0x19213380, decoded.pulseLogEntry[43])
+            XCTAssertEqual(0x1c203180, decoded.pulseLogEntry[44])
+            XCTAssertEqual(0x21203280, decoded.pulseLogEntry[45])
+            XCTAssertEqual(0x24213200, decoded.pulseLogEntry[46])
+            XCTAssertEqual(0x29203500, decoded.pulseLogEntry[47])
+            XCTAssertEqual(0x2c213100, decoded.pulseLogEntry[48])
+            XCTAssertEqual(0x31213400, decoded.pulseLogEntry[49])
         } catch (let error) {
             XCTFail("message decoding threw error: \(error)")
         }
     }
 
-    func testPodInfoFlashLogPrevious() {
+    func testPodInfoPulseLogPrevious() {
         //02 cb 51 0032 14602500 19612800 1c612400 21612800 24612500 29612900 2c602600 31602a00 34602600 39612a80 3c612680 41602c80 00602780 05632880 08602580 0d612880 10612580 15612780 18602380 1d602680 20612280 25602700 28612400 2d212800 30202700 35202a00 38202700 3d202a00 40202900 45202c00 48202a00 4d212c00 50212900 55212c00 58212980 5d202b80 60202880 01202d80 04212a80 09202d80 0c212980 11212a80 14212980 1921801c 212a8021 212c8024 202c0029 212f002c 212d0031 20310082
         do {
             // Decode
-            let decoded = try PodInfoFlashLogPrevious(encodedData: Data(hexadecimalString: "51003214602500196128001c6124002161280024612500296129002c60260031602a003460260039612a803c61268041602c800060278005632880086025800d6128801061258015612780186023801d6026802061228025602700286124002d2128003020270035202a00382027003d202a004020290045202c0048202a004d212c005021290055212c00582129805d202b806020288001202d8004212a8009202d800c21298011212a80142129801921801c212a8021212c8024202c0029212f002c212d003120310082")!)
-            XCTAssertEqual(.dumpOlderFlashlog, decoded.podInfoType)
+            let decoded = try PodInfoPulseLogPrevious(encodedData: Data(hexadecimalString: "51003214602500196128001c6124002161280024612500296129002c60260031602a003460260039612a803c61268041602c800060278005632880086025800d6128801061258015612780186023801d6026802061228025602700286124002d2128003020270035202a00382027003d202a004020290045202c0048202a004d212c005021290055212c00582129805d202b806020288001202d8004212a8009202d800c21298011212a80142129801921801c212a8021212c8024202c0029212f002c212d003120310082")!)
+            XCTAssertEqual(.dumpOlderPulseLog, decoded.podInfoType)
             XCTAssertEqual(50, decoded.nEntries)
-            XCTAssertEqual(Data(hexadecimalString:"14602500196128001c6124002161280024612500296129002c60260031602a003460260039612a803c61268041602c800060278005632880086025800d6128801061258015612780186023801d6026802061228025602700286124002d2128003020270035202a00382027003d202a004020290045202c0048202a004d212c005021290055212c00582129805d202b806020288001202d8004212a8009202d800c21298011212a80142129801921801c212a8021212c8024202c0029212f002c212d003120310082"), decoded.hexWordLog)
+            XCTAssertEqual(0x14602500, decoded.pulseLogEntry[0])
```

Comment:
If the same code is decoding entry[0] as entry[n], it's probably sufficient to just test any variations that exercise different aspects of the decoding.